src/plugin.js

```diff
@@ -309,9 +324,17 @@ const plugin = function(options) {
   // because it doesn't make sense to pass it to each Overlay component.
   delete settings.overlays;
 
-  this.overlays_ = overlays.map(
-    o => this.addChild('overlay', videojs.mergeOptions(settings, o))
-  );
+  this.overlays_ = overlays.map(o => {
+    let mergeOptions = videojs.mergeOptions(settings, o);
+
+    // Attach bottom aligned overlays to the control bar so
+    // they will adjust positioning when the control bar minimizes
+    if (this.controlBar && mergeOptions.align.indexOf('bottom') !== -1) {
```

Comment:
If someone is relying on bottom-aligned overlays with a custom control bar, they will potentially no longer see these overlays because it's possible they just hide the control bar using `display: none` rather than disable it in the config.